sljit_src/sljitNativeLOONGARCH_64.c

```diff
@@ -2972,7 +2974,13 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_atomic_load(struct sljit_compiler
 		mem_reg = TMP_REG2;
 	}
 
```

Comment:
TMP_REG3 is reused in two independent code paths, because mem_unaligned and (GET_OPCODE(op) == SLJIT_MOV_32) are mutually exclusive.

SLJIT_ASSERT(mem_reg != dst) would work but it is a little indirect, and didn't want to add an assert with another call to GET_OPCODE(), neither do the suggested "optimization" above